ci/build-unix.yml

```diff
@@ -40,3 +44,20 @@ steps:
     env:
       # to connect to bintray
       JFROG_CONFIG_CONTENT: $(JFROG_CONFIG_CONTENT)
+  - bash: |
+      set -euxo pipefail
+      git tag v$(release_tag)
```

Comment:
Is there a reason to make a lightweight tag here rather than an annotated tag? Quoting `man git-tag`:
```
       Annotated tags are meant for release while lightweight tags are meant
       for private or temporary object labels.
```

Also please update the release instructions in `release/RELEASE.md` and remove the tagging step.

Reply:
Main reason was I did not have any useful additional information to put in the tag, so I did not see the benefit. I'll mirror the instructions in RELEASE.md.